programs/tensorflowDetection2D/MainDetector.cpp

```diff
@@ -206,9 +214,11 @@ int maindetector::detect(std::string labels, std::string graph, std::string vide
         yarp::sig::ImageOf<yarp::sig::PixelBgr> C;
         C.setExternal(frame.data,frame.size[1],frame.size[0]);
         sender_port_post.write(C);
-        cv::imshow("Video source: Processed", frame);
+  cv::imshow("Video source: Processed", frame);
         cv::waitKey(5);
```

Comment:
Can we remove all traces of OpenCV?

Reply:
Right now, not all, i´m trying to remove OpenCV and use YARP instead, but `cv::imshow` it´s going to be removed. I´m using it for testing.